third_party/openthread/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ set(OT_BUILD_EXECUTABLES OFF CACHE BOOL "disable building executables" FORCE)
 set(OT_BUILTIN_MBEDTLS_MANAGEMENT OFF CACHE BOOL "diable mbedTLS management" FORCE)
 set(OT_COMMISSIONER ON CACHE BOOL "enable commissioner" FORCE)
 set(OT_DAEMON ON CACHE BOOL "enable daemon mode" FORCE)
-set(OT_FIREWALL ON CACHE BOOL "enable firewall feature" FORCE)
+set(OT_FIREWALL ON CACHE BOOL "enable firewall feature")
```

Comment:
Please document why we need to remove `FORCE`. Otherwise, it will look more like a mistake :)